File: src/main/java/org/zenbot/timetableupdater/domain/BusArrival.java
package org.zenbot.timetableupdater.domain;

import lombok.Data;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

@Data
@Document
public class BusArrival {

    @Id
    private String id;
    private boolean lowfloor;
    private int arrivalMinute;
}

File: src/main/kotlin/org/zenbot/timetableupdater/domain/BusArrival.kt
package org.zenbot.timetableupdater.domain

import org.springframework.data.annotation.Id
import org.springframework.data.mongodb.core.mapping.Document

@Document
data class BusArrival(
        @Id var id: String? = null,
        var lowfloor: Boolean = false,
        var arrivalHour: Int = 0,
        // Might be null as there could be no arrival in a particular hour
        var arrivalMinute: Int? = null
)